code/_helpers/unsorted.dm

```diff
@@ -608,15 +602,25 @@ proc/GaussRandRound(var/sigma,var/roundto)
 	return round(GaussRand(sigma),roundto)
 
 //Will return the contents of an atom recursivly to a depth of 'searchDepth'
-/atom/proc/GetAllContents(searchDepth = 5)
-	var/list/toReturn = list()
-
-	for(var/atom/part in contents)
-		toReturn += part
-		if(part.contents.len && searchDepth)
-			toReturn += part.GetAllContents(searchDepth - 1)
-
-	return toReturn
+/atom/proc/GetAllContents(var/T)
```

Comment:
If you absolutely must change this proc, at least change it to /tg/'s version [here](https://github.com/tgstation/tgstation/blob/master/code/__HELPERS/unsorted.dm) that avoids unnecessary list creation and calls to Cut().